edge-apps/countdown-timer/screenly.yml

```diff
@@ -40,13 +40,13 @@ settings:
   tag_manager_id:
     type: string
     default_value: GTM-P98SPZ9Z
-    title: tag_manager_id
+    title: Tag Manager ID
     optional: true
     help_text: Specify a Google Tag Manager ID
     is_global: true
   target_timestamp:
     type: string
     default_value: 2025-01-01T00:00:00
-    title: target_timestamp
+    title: Target Countdown Date & Time
```

Comment:
```suggestion
    title: Target Countdown Datetime
```

Reply:
I'd just rephrase it as 'Countdown Target'